[2025-02-17T20:19:00.443+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-17T20:19:00.642+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: name_of_pipeline.parser_1 scheduled__2023-06-11T00:00:00+00:00 [queued]>
[2025-02-17T20:19:00.722+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: name_of_pipeline.parser_1 scheduled__2023-06-11T00:00:00+00:00 [queued]>
[2025-02-17T20:19:00.725+0000] {taskinstance.py:2867} INFO - Starting attempt 2 of 1
[2025-02-17T20:19:00.925+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): parser_1> on 2023-06-11 00:00:00+00:00
[2025-02-17T20:19:00.998+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=2453) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-17T20:19:01.031+0000] {standard_task_runner.py:72} INFO - Started process 2484 to run task
[2025-02-17T20:19:01.025+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'name_of_pipeline', 'parser_1', 'scheduled__2023-06-11T00:00:00+00:00', '--job-id', '757', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp05u9jw48']
[2025-02-17T20:19:01.036+0000] {standard_task_runner.py:105} INFO - Job 757: Subtask parser_1
[2025-02-17T20:19:01.427+0000] {task_command.py:467} INFO - Running <TaskInstance: name_of_pipeline.parser_1 scheduled__2023-06-11T00:00:00+00:00 [running]> on host f4228ad55513
[2025-02-17T20:19:01.883+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='name_of_pipeline' AIRFLOW_CTX_TASK_ID='parser_1' AIRFLOW_CTX_EXECUTION_DATE='2023-06-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-06-11T00:00:00+00:00'
[2025-02-17T20:19:01.921+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-17T20:19:01.923+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-17T20:19:01.925+0000] {logging_mixin.py:190} INFO - Current task name:parser_1 state:running start_date:2025-02-17 20:19:00.650849+00:00
[2025-02-17T20:19:01.925+0000] {logging_mixin.py:190} INFO - Dag name:name_of_pipeline and current dag run status:running
[2025-02-17T20:19:01.927+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-17T20:19:01.933+0000] {pipeline.py:14} INFO - Starting parser 1
[2025-02-17T20:19:01.937+0000] {logging_mixin.py:190} INFO - Парсер 1 выполняется
[2025-02-17T20:19:01.940+0000] {python.py:240} INFO - Done. Returned value was: Данные от парсера 1
[2025-02-17T20:19:03.705+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-17T20:19:03.713+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=name_of_pipeline, task_id=parser_1, run_id=scheduled__2023-06-11T00:00:00+00:00, execution_date=20230611T000000, start_date=20250217T201900, end_date=20250217T201903
[2025-02-17T20:19:03.825+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-17T20:19:03.834+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-17T20:19:03.863+0000] {logging_mixin.py:190} INFO - Dag name:name_of_pipeline queued_at:2025-02-17 20:18:33.664430+00:00
[2025-02-17T20:19:03.865+0000] {logging_mixin.py:190} INFO - Task hostname:f4228ad55513 operator:PythonOperator
[2025-02-17T20:19:03.944+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-17T20:19:04.367+0000] {taskinstance.py:3901} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2025-02-17T20:19:04.379+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-17T20:19:11.929+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-17T20:19:12.018+0000] {taskinstance.py:2604} INFO - Dependencies not met for <TaskInstance: name_of_pipeline.parser_1 scheduled__2023-06-11T00:00:00+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2025-02-17T20:19:12.024+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
